src/applications/personalization/dashboard/reducers/folders.js

```diff
@@ -1,5 +1,5 @@
-import _ from 'lodash';
-import set from 'lodash/fp/set';
+import kebabCase from 'lodash/kebabCase';
+import set from 'platform/utilities/data/set';
```

Comment:
the usage of our platform `set` is the same as lodashfp's `set`?

Reply:
I think the function signature is different

Reply:
Hmm there are a couple places in this PR where the swap is made but no change to usage. Can you double check that @theodur ?

Reply:
The function signature for lodash's set is different, but the signature for [lodash/fp's set](https://gist.github.com/jfmengels/6b973b69c491375117dc#_setpath-value-object) is the same as the one we use in platform.

Reply:
Thanks for confirming! I would hope unit tests would have failed if this wasn’t the case, but still.